packages/collector/test/tracing/frameworks/express_uncaught_errors/app.js

```diff
@@ -13,6 +13,7 @@ process.on('SIGTERM', () => {
   process.exit(0);
```

Comment:
Q: Putting it here. I was wondering why we don't add a full express v5 app here (`app.js` and `app_v5.js`)? We are using the epxress v4 syntax still for testing against the beta version. I assume you have decided against because it does not bring any advantages? Or why did you decide like this? Can you explain your thinking please?

Reply:
Thank you for your question. I chose not to include a full Express v5 app in our setup for the following reasons:
1. Since v5 is still in beta and may introduce changes, there are no immediate advantages to adopting v5 syntax at this time, particularly as there have been no substantial changes in the beta syntax that impact our existing app.
2. Sticking with Express v4 syntax allows us to transition gradually to v5 once it is officially released and stable, which simplifies maintenance by focusing on one syntax version at a time.